Specify comparison for Track Name / Namespace (#329)
moq-wg/moq-transport: draft-ietf-moq-transport.md

~~~

In this specification, both the Track Namespace and the Track Name are
not constrained to a specific type of encoding. They carry a range of
bytes and any comparison between two Track Namespaces/Track Names is
done simply by exact compare of the bytes. Higher level specifications

> may constrain the information in these fields to UTF-8 or portions of
> particular types or URLs. Any specification that does that needs to

Reviewer: ```suggestion
may constrain the information in these fields, for example by restricting
them to UTF-8. Any specification that does that needs to
```

specify the canonicalization into the bytes in the Track Namespace or

> Track Name such that comparison works correctly.

Reviewer: ```suggestion
Track Name such that exact comparison works.
```
